src/main/java/bio/terra/pdao/bigquery/BigQueryPdao.java

```diff
@@ -248,10 +248,19 @@ public void createDataset(bio.terra.metadata.Dataset dataset, List<String> rowId
     }
 
     @Override
-    public void addReaderGroupToDataset(bio.terra.metadata.Dataset dataset, String readersEmail) {
+    public void addReaderGroupToDataset(bio.terra.metadata.Dataset dataset, String readerPolicyGroupEmail) {
         BigQueryProject bigQueryProject = bigQueryProjectForDataset(dataset);
         bigQueryProject.addDatasetAcls(dataset.getName(),
-            Collections.singletonList(Acl.of(new Acl.Group(readersEmail), Acl.Role.READER)));
+            Collections.singletonList(Acl.of(new Acl.Group(readerPolicyGroupEmail), Acl.Role.READER)));
+    }
+
+    @Override
+    public void grantReadAccessToStudy(Study study, List<String> policyGroupEmails) {
+        BigQueryProject bigQueryProject = bigQueryProjectForStudy(study);
+        bigQueryProject.addDatasetAcls(prefixName(study.getName()), policyGroupEmails
```

Comment:
Nit: I'm not a fan of embedded streams like that. Perhaps build the collection first and then make the add call?